src/App/Fossa/Init.hs

```diff
@@ -0,0 +1,102 @@
+{-# LANGUAGE TemplateHaskell #-}
+
+module App.Fossa.Init (
+  initCommand,
+
+  -- * testing only
+  mkFossaYml,
+  mkFossaDeps,
+  exampleFossaYml,
+  exampleFossaDeps,
+) where
+
+import Control.Algebra (Has)
+import Control.Carrier.Diagnostics (Diagnostics, logWithExit_)
+import Control.Carrier.Lift (sendIO)
+import Control.Carrier.Stack (runStack)
+import Control.Carrier.Telemetry (withTelemetry)
+import Control.Effect.Lift (Lift)
+import Data.ByteString (ByteString)
+import Data.ByteString qualified as BS
+import Data.FileEmbed.Extra (embedFileIfExists)
+import Effect.Logger (Logger, Severity (SevInfo), logInfo, pretty, withDefaultLogger)
+import Effect.ReadFS (ReadFS, doesFileExist, getCurrentDir, runReadFSIO)
+import Options.Applicative (CommandFields, Mod, Parser, info, progDesc)
+import Options.Applicative.Builder (command)
+import Path (
+  Abs,
+  Dir,
+  File,
+  Path,
+  Rel,
+  mkRelFile,
+  toFilePath,
+  (</>),
+ )
+
+initCommand :: Mod CommandFields (IO ())
+initCommand = command "init" (info run $ progDesc "Creates example .fossa.yml and fossa-deps.yml file")
+  where
+    run :: Parser (IO ())
+    run = pure $ withTelemetry . runStack . withDefaultLogger SevInfo . logWithExit_ . runReadFSIO $ runInit
+
+runInit :: (Has Diagnostics sig m, Has (Lift IO) sig m, Has ReadFS sig m, Has Logger sig m) => m ()
+runInit = do
+  dir <- getCurrentDir
+  mkFossaYml dir
+  mkFossaDeps dir
+
+mkFossaYml :: (Has Diagnostics sig m, Has (Lift IO) sig m, Has ReadFS sig m, Has Logger sig m) => Path Abs Dir -> m ()
+mkFossaYml baseDir = do
+  (fossaYml, fossaYmlExist) <- checkFile (baseDir </> fossaYmlFile)
```

Comment:
I know this wasn't in the ACs of the ticket, but for broker we made `broker init` also output a `.example` file. In this case it would be `.fossa.yml.example`. 

This contains exactly the same contents as the generated .fossa.yml. The benefit is that the user has the unchanged version for reference while they are editing the non-example version.

We also always overwrite the example version with the current contents. This allows us to show the user new features even there is an existing .fossa.yml or fossa-deps.yml.